package.json

```diff
@@ -50,7 +50,7 @@
     "test-integration-dev": "turbo run test-integration-dev",
     "lint": "eslint . --ext .ts,.js",
     "prepare": "husky install",
-    "pack": "cd utils && node -r ts-eager/register ./pack.ts"
+    "pack": "cd ./api/_lib/script && node -r ts-eager/register ./pack.ts"
```

Comment:
Can we remove this script from package.json if we make the "pack" step part of the `_lib/script/build.ts`?

Reply:
Yes, but these steps were intentionally separated, cc @TooTallNate